src/Bundle/Extractor/DoctrineTypes.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Zenstruck\Foundry\Bundle\Extractor;
+
+class DoctrineTypes
+{
+    public const DOCTRINE_TYPES = [
+        'ARRAY' => '[]',
+        'ASCII_STRING' => 'self::faker()->text()',
+        'BIGINT' => 'self::faker()->randomNumber()',
+        'BINARY' => '// @TODO add Binary manualy',
```

Comment:
Actually, let's remove the unsupported types from this array and in `Property::createFakerMethodFromDoctrineType()`, when the type is not found, generate a todo.